src/sentry/uptime/endpoints/organization_uptime_summary.py

```diff
@@ -277,7 +248,7 @@ def _format_response(
 
             # Get avg_duration_us if available, otherwise set to None
             avg_duration_us = None
-            if include_avg_duration and "avg_duration_us" in row_dict:
+            if "avg_duration_us" in row_dict:
```

Comment:
We probably don't need this check anymore. I think it's always going to be in the dict now?